Arduino/McLighting/definitions.h

```diff
@@ -52,35 +55,36 @@ const char HOSTNAME[] = "McLighting01";   // Friedly hostname
 #endif
 
 // parameters for automatically cycling favorite patterns
-uint32_t autoParams[][4] = { // color, speed, mode, duration (seconds)
-  {0xff0000, 200,  1,  5.0}, // blink red for 5 seconds
-  {0x00ff00, 200,  3, 10.0}, // wipe green for 10 seconds
-  {0x0000ff, 200, 11,  5.0}, // dual scan blue for 5 seconds
-  {0x0000ff, 200, 42, 15.0}  // fireworks for 15 seconds
+uint32_t autoParams[][4] = {  // color, speed, mode, duration (milliseconds)
```

Comment:
Verry much approve this. :+1:

Reply:
Contribution by @dale3h https://github.com/toblum/McLighting/commit/67a98c28b899dbb608bd32da291e3e250e1f41ff